backend/src/test/java/com/happy/friendogly/club/service/ClubCommandServiceTest.java

```diff
@@ -220,4 +256,79 @@ void deleteClubMember_WhenIsEmptyDelete() {
 
         assertThat(clubRepository.findById(savedClub.getId()).isEmpty()).isTrue();
     }
+
+    @DisplayName("참여 중인 회원을 삭제하고, 만약 Full 상태면 Open으로 바꾼다.")
```

Comment:
```suggestion
    @DisplayName("Full 상태에서 회원을 삭제하면 Open으로 바꾼다.")
```

처음 봤을 때 이해가 잘 안 되었어요 !!!

Reply:
넵!!